### PR TITLE
CP tower armour change

### LINKED ISSUE #2144: CP Armour Change - Unarmoured to Heavy
Change Control Point armour to Heavy.
Change Elune's Grace to resist Magic instead of Peirce, 0.2, 0.2.
Rename to Magic Resistance.
Change in GameSetup

### PR DESCRIPTION
**Pull request by Technopig1992**
Closes #2144 

Changed CP Armour  Unarmoured>Heavy.
Changed Piercing Resistance name to Magic Resistance, now resists 80% of magic attacks. 
Moved Magic Resistance Icon location 1,1 >1,2

Misc
Replaced Flightpath in setup with correct Constant. 
Replaced Biodomes in setup with correct Constant.
Replaced ArcaneAbsorption in setup with correct Constant.

### CODE REVIEW

**Comment by YakaryBovine** [comment]
I'll check this out tomorrow morning. It will also wait until another bug fix release is out.

File: mapdata/WarcraftLegacies/AbilityData/Skin/1479749953.json [hunk 1]
```diff
@@ -29,7 +25,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Reduces the damage taken from Piercing attacks to \u003CA13X,DataA1,%\u003E%."
+      "Value": "Reduces the damage taken from Magic attacks by \u003CA13X,DataE1,%\u003E%."
```
Comment (YakaryBovine) [inline mapdata/WarcraftLegacies/AbilityData/Skin/1479749953.json:28]: I believe this tooltip is incorrect. The ability you have reduces magic damage _to_ 80%, not by.

**Review by YakaryBovine — CHANGES_REQUESTED** [review]
Just one change needed.

File: mapdata/WarcraftLegacies/AbilityData/Skin/1479749953.json (hunk 1 above)
Comment (Technopig1992) [reply mapdata/WarcraftLegacies/AbilityData/Skin/1479749953.json]: ty changed

**Review by YakaryBovine — APPROVED** [review]
Looks good. Put the change in here please: https://docs.google.com/document/d/112XVlOtW3A1CaFIZxaYmbxBmjL5T_qNFREsNPYPBv4U/edit#heading=h.w045veyw5eun

**Pull request merged**